samples/java-eventsourced-customer-registry-subscriber/src/main/proto/customer/view/customer_view.proto

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 Lightbend Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package customer.view;
+
+option java_outer_classname = "CustomerViewModel";
+
+import "customer/api/customer_api.proto";
+import "customer/api/publisher_api.proto";
+import "kalix/annotations.proto";
+import "google/protobuf/empty.proto";
+
+import "google/api/annotations.proto";
+
+// tag::view[]
+service AllCustomersView {
+  option (kalix.codegen) = {
+    view: {}
+  };
+
+  // consume events published by scala-eventsourced-customer-registry/CustomerEventsServiceAction
+  option (kalix.service).eventing.in.direct = { // <1>
+    // end::view[]
+    // Normally the Kalix service deployment name
+    // (but localhost + hardcoded port here to be able to run both locally)
+    service: "localhost:9000"
```

Comment:
will this become a problem if users download the source and try to deploy it?

Reply:
maybe not a problem since it's not a quickstart sample (yet)

Reply:
Yes, the would have to change it to deploy (but changing that's easier than changing the other things to get it running locally, with the proxy port etc).